python/pacemaker/_cts/environment.py

```diff
@@ -300,16 +300,14 @@ def _parse_args(self, argv):
         parser = argparse.ArgumentParser(epilog=f"{sys.argv[0]} -g virt1 -r --stonith ssh --schema pacemaker-2.0 500")
 
         grp1 = parser.add_argument_group("Common options")
-        grp1.add_argument("-g", "--dsh-group", "--group",
```

Comment:
`cts/cts` uses this, actually. Perhaps it should be dropped there too, but that needs some more examination.

Don't focus too much on this commit.